src/main/features/response/form/models/statement-of-means/debtRow.ts

```diff
@@ -0,0 +1,73 @@
+import { IsDefined, Min, ValidateIf } from 'class-validator'
+
+import { ValidationErrors as GlobalValidationErrors } from 'forms/validation/validationErrors'
+import { MultiRowFormItem } from 'forms/models/multiRowFormItem'
+import { Fractions } from 'forms/validation/validators/fractions'
+import { toNumberOrUndefined } from 'common/utils/numericUtils'
+import { IsNotBlank } from 'forms/validation/validators/isBlank'
+import { MaxLength } from 'forms/validation/validators/maxLengthValidator'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+
+export class ValidationErrors {
+  static readonly DEBT_REQUIRED: string = 'Enter a value'
+}
+
+export class DebtRow extends MultiRowFormItem {
+
+  @ValidateIf(o => o.isAtLeastOneFieldPopulated())
+  @IsDefined({ message: ValidationErrors.DEBT_REQUIRED })
+  @IsNotBlank({ message: ValidationErrors.DEBT_REQUIRED })
+  @MaxLength(ValidationConstraints.STANDARD_TEXT_INPUT_MAX_LENGTH, { message: GlobalValidationErrors.TEXT_TOO_LONG })
+  debt?: string
+
+  @ValidateIf(o => o.isAtLeastOneFieldPopulated())
+  @IsDefined({ message: GlobalValidationErrors.NUMBER_REQUIRED })
+  @IsDefined({ message: GlobalValidationErrors.NUMBER_REQUIRED })
+  @Fractions(0, 2, { message: GlobalValidationErrors.AMOUNT_INVALID_DECIMALS })
+  @Min(0, { message: GlobalValidationErrors.POSITIVE_NUMBER_REQUIRED })
```

Comment:
we have `DefaultValidationErrors` and `GlobalValidationErrors` now :(
should just rename the class imo, not required for this change..